modules/oe_theme_helper/tests/src/Kernel/ExternalLinksTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme_helper\Kernel;
+
+use Drupal\Core\Url;
+use Drupal\Tests\oe_theme\Kernel\AbstractKernelTestBase;
+
+/**
+ * Provides test coverage for the ExternalLinks service.
+ *
+ * @group batch1
+ */
+class ExternalLinksTest extends AbstractKernelTestBase {
+
+  /**
+   * Covers isExternalLink method.
+   */
+  public function testIsExternalLink(): void {
```

Comment:
Let's also add some tests asserting to `true`, if we don't do this we might as well have a method that always returns false, for any combinations of inputs.